examples/using-styled-components/package.json

```diff
@@ -5,22 +5,22 @@
   "version": "2.0.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
-    "babel-plugin-styled-components": "^1.8.0",
-    "gatsby": "^2.0.0",
-    "gatsby-plugin-google-analytics": "^2.0.5",
-    "gatsby-plugin-offline": "^2.0.5",
-    "gatsby-plugin-react-helmet": "^3.0.0",
-    "gatsby-plugin-styled-components": "^3.0.0",
-    "gatsby-plugin-typography": "^2.2.0",
-    "lodash": "^4.17.10",
-    "react": "^16.4.0",
-    "react-dom": "^16.4.0",
+    "babel-plugin-styled-components": "^1.10.0",
+    "gatsby": "^2.0.91",
```

Comment:
Because we don't use lock files for these examples--these are synonymous. e.g. `^2.0.0` will resolve to the latest version of gatsby, which in this case is now `2.0.93`.

So while this change is appreciated--it's not necessary! This same comment applies to most of these changes!

Reply:
Oh, ok. Didn't know about that actually :smile: